pxtcompiler/emitter/hexfile.ts

```diff
@@ -646,15 +646,17 @@ ${hex.hexPrelude()}
     }
 
     function mkProcessorFile(nativeType: string) {
-        let processor: assembler.AbstractProcessor = null
+        let b: assembler.File
+
         if (nativeType == NATIVE_TYPE_AVR)
```

Comment:
use isAvr() ?

Reply:
isAvr() == AVR || AVRVM - this is only for native AVR